app/styles/app/layouts/repo.scss

```diff
@@ -24,20 +24,31 @@
       margin-right: 10px;
     }
   }
-
+  
   .repo-title {
-    margin: 0;
-    font-size: 36px;
-    font-weight: $font-weight-normal;
-
-    a {
-      color: $asphalt-grey;
-      border-bottom: 2px solid transparent;
-      transition: border-color 100ms ease;
-
-      &:hover,
-      &:active {
-        border-color: $asphalt-grey;
+    margin-right: 1.2em;
+
+    @media #{$small-only} {
+      margin-bottom: 0.5em;
+      h1 {
+        font-size: 24px!important;
```

Comment:
I try to avoid use of `!important` whenever possible. It's a big jump in specificity that could make the codebase harder to maintain. 

An easy way to address this could be to move this media query below the `h1` definition a few lines down, since stuff defined later takes precedence. In this case you could split it into 2 separate media queries or keep it as 1, whichever makes sense to you.

You could also add a class to raise the specificity of this definition without resorting to `!important`, but that seems a little weirder to me.

Reply:
Oh here are some links on this issue:
https://uxengineer.com/css-specificity-avoid-important-css/
https://css-tricks.com/strategies-keeping-css-specificity-low/